xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -108,7 +108,7 @@
                   "album", "albumtype", "genre", "country", "year", "rating", "userrating", "votes", "top250", "programcount",
                   "playlist", "episode", "season", "totalepisodes", "watchedepisodes", "tvshowstatus", "tvshowtitle",
                   "sorttitle", "productioncode", "mpaa", "studio", "dateadded", "lastplayed", "playcount", "listeners",
-                  "bitrate", "random", "totaldiscs", "originalyear", "bpm" ]
+                  "bitrate", "random", "totaldiscs", "originalyear", "bpm", "samplerate" ]
```

Comment:
No "samplerate" sorting, think this got left, but this line does need a fix to  "originaldate"
`"bitrate", "random", "totaldiscs", "originaldate", "bpm" ]`
As our first JSON amending PR this is a good place to fix that :)

Reply:
Fixed.